lib.php

```diff
@@ -416,23 +416,20 @@ function tincanlaunch_pluginfile($course, $cm, $context, $filearea, $args, $forc
     require_login($course, true, $cm);
     $canmanageactivity = has_capability('moodle/course:manageactivities', $context);
 
+    $filename = array_pop($args);
+    $filepath = implode('/', $args);
     if ($filearea === 'content') {
-        $filename = array_pop($args);
-        $filepath = implode('/', $args);
         $lifetime = null;
     } else if ($filearea === 'package') {
-        $relativepath = implode('/', $args);
-        $fullpath = "/$context->id/tincanlaunch/package/0/$relativepath";
```

Comment:
It doesn't _seem_ too--in fact, if you look at `master`, `$fullpath` is never used. This looks to be copied from the SCORM module, but the SCORM module uses this value for calculating the SHA1 to look up the file (which this module does not do).